dap-mode.el

```diff
@@ -739,16 +740,19 @@ thread exection but the server will log message."
 (defun dap--insert-at-point-max (str)
   "Inserts STR at point-max of the buffer."
   (goto-char (point-max))
-  (insert str))
+  (insert (ansi-color-apply str)))
 
 (defun dap--print-to-output-buffer (debug-session str)
   "Insert content from STR into the output buffer associated with DEBUG-SESSION."
   (with-current-buffer (get-buffer-create (dap--debug-session-output-buffer debug-session))
+    (font-lock-mode t)
+    (setq-local buffer-read-only nil)
```

Comment:
Why not use a let binding here ?

Reply:
Because by default it's not set, and we make it read-only with `(setq-local buffer-read-only t)` below. If we use let binding we will still need this line, so it doesn't make much sense.